docs/src/main/asciidoc/opentelemetry.adoc

```diff
@@ -243,15 +243,16 @@ The https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/ma
 implementation("io.opentelemetry.instrumentation:opentelemetry-jdbc")
 ----
 
-As it uses a dedicated JDBC driver, you must configure your datasource and Hibernate ORM to use it.
+As it uses a dedicated JDBC driver, you must enable JDBC tracing for your datasource:
```

Comment:
As it uses a dedicated Datasource wrapper, you must enable JDBC tracing for your datasource:

Reply:
Thanks. Now I also mentioned `JDBC driver`, that it not true anymore.